src/controllers/mysql/queries.ts

```diff
@@ -40,12 +40,11 @@ export function createPost(postObject: PostObject){
 }
 
 //Needs to have the correct FROM destination
+//Currently just finds the highest (max) ID and returns it, might need to be readjusted in the future.
 export function latestDigestedPostNumber() {
   return new Promise((resolve) => {
-    connection.query('SELECT COUNT(*) as total FROM post', (error, results, fields) => {
-      let latestDigestedNumber = results[0].total;
-      
-      //console.log(Number(latestDigestedNumber));
+      connection.query('SELECT MAX(id) as id FROM post', (error, results, fields) => {
```

Comment:
Change to select x newest (example):
SELECT * FROM dolphin ORDER BY id DESC LIMIT 5

Reply:
Made a minor adjustment to the suggested query:
`SELECT * FROM dolphin ORDER BY id DESC LIMIT 1`
Since we are only interested in returning the latest (single) digestedID. Though we still need to point to the right identification (the hassnet_id), but won't do that until the DB is setup correctly.

Reply:
Tried to implement the suggested Query, but it breaks/fails the backend.

Reply:
Fixed the suggested Query-statement:
`SELECT * FROM user ORDER BY id DESC LIMIT 1`